src/function_lua.c

```diff
@@ -462,9 +462,6 @@ int luaEngineInitEngine() {
     lua_pushcfunction(lua_engine_ctx->lua, luaRegisterFunction);
     lua_settable(lua_engine_ctx->lua, -3);
 
-    luaRegisterLogFunction(lua_engine_ctx->lua);
-    luaRegisterVersion(lua_engine_ctx->lua);
```

Comment:
@soloestoy those are meant to register it for the library load context. On library load you do not have the same globals as when running a function.

Reply:
OK, get it